pkgs/development/interpreters/perl/intepreter.nix

```diff
@@ -110,7 +113,7 @@ stdenv.mkDerivation (rec {
     ++ lib.optional stdenv.isSunOS "-Dcc=gcc"
     ++ lib.optional enableThreading "-Dusethreads"
     ++ lib.optional (!enableCrypt) "-A clear:d_crypt_r"
-    ++ lib.optional stdenv.hostPlatform.isStatic "--all-static"
+    ++ lib.optionals stdenv.hostPlatform.isStatic [ "--all-static" "-Uusedl" ]
```

Comment:
I'm not very familiar with this flag, but it seems to be used upstream and produces a binary in the end.